### PR TITLE
Added doc comments.

### PR DESCRIPTION
**Pull request by anatoo**
* Added @extensionOf tags.
 * Added @initonly tags.
 * Added doc comments about ons.GestureDetector.
 * Fixed some trivial doc comment issues.

### CODE REVIEW

File: framework/js/gestureDetector.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+ */
+
+/**
+ * @ngdoc method
+ * @signature dispose()
+ * @description
+ *  [en][/en]
+ *  [ja]このインスタンスでのジェスチャの検知や、イベントハンドラを全て解除して廃棄します。[/ja]
+ */
+
```
Comment (anatoo) [inline framework/js/gestureDetector.js]: @argelius Please add English transition comments on ons.GestureDetector.

**Comment by anatoo** [comment]
@argelius  Merge this changes if there is no problems.

**Comment by argelius** [comment]
Thanks @anatoo!

I will merge this now.

**Comment by anatoo** [comment]
:+1:

**Pull request merged**